Client/mods/deathmatch/logic/lua/CLuaFunctionDefs.World.cpp

```diff
@@ -1764,6 +1764,263 @@ int CLuaFunctionDefs::GetBirdsEnabled(lua_State* luaVM)
     return 1;
 }
 
+int CLuaFunctionDefs::RemoveBirds(lua_State* luaVM)
+{
+    if (CStaticFunctionDefinitions::RemoveBirds())
+    {
+        lua_pushboolean(luaVM, true);
+        return 1;
+    }
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
+int CLuaFunctionDefs::AddBirds(lua_State* luaVM)
+{
+    //  bool addBirds ( float startX, float startY, float startZ, float destX, float destY, float destZ, [int birdsCount = 1, int birdType = 0, bool checkObstacles = false] )
+
+    CVector          vecStartPosition, vecDestPosition;
+    int              iNumBirds, iBirdType;
+    bool             bCheckObstacles;
+    CScriptArgReader argStream(luaVM);
+
+    argStream.ReadVector3D(vecStartPosition);
+    argStream.ReadVector3D(vecDestPosition);
+    argStream.ReadNumber(iNumBirds, 1);
+    argStream.ReadNumber(iBirdType, 0);
+    argStream.ReadBool(bCheckObstacles, false);
+
+    if (!argStream.HasErrors())
+    {
+        int iBirdsCreated = CStaticFunctionDefinitions::AddBirds(vecStartPosition, vecDestPosition, iNumBirds, iBirdType, bCheckObstacles);
+        if (iBirdsCreated != 0)
+        {
+            lua_pushnumber(luaVM, iBirdsCreated);
+            return 1;
+        }
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
+
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
+int CLuaFunctionDefs::ResetTimeCycle(lua_State* luaVM)
+{
+    if (g_pGame->GetTimeCycle()->ResetTimeCycle())
+    {
+        lua_pushboolean(luaVM, true);
+        return 1;
+    }
+    lua_pushboolean(luaVM, false);
+    return 1;
+}
+
+int CLuaFunctionDefs::SetTimeCycle(lua_State* luaVM)
+{
+    //  bool setTimeCycle ( string property, [float value = -1, int weatherID = -1, int cycle = -1] )
+
+    SString          strPropertyName;
+    float            fValue;
+    int              iWeatherID, iCycleNum;
+    CScriptArgReader argStream(luaVM);
+
+    argStream.ReadString(strPropertyName);
+    if (argStream.NextIsNone())
+    {
+        argStream.Skip(1);
+        fValue = -1;
+    }
+    else if (argStream.NextIsBool())
+    {
+        bool bValue;
+        argStream.ReadBool(bValue, false);
+        if (!bValue)
+            fValue = -1;
+        else
+            argStream.SetCustomError("The value should be either a number or false");
+    }
+    else
+    {
+        argStream.ReadNumber(fValue, -1, false);
+    }
+    argStream.ReadNumber(iWeatherID, -1, false);
+    argStream.ReadNumber(iCycleNum, -1, false);
+
+    if (iWeatherID < -1 || iWeatherID > 23)
+        argStream.SetCustomError("The weatherID should be between 0 and 23");
+    if (iCycleNum < -1 || iCycleNum > 7)
+        argStream.SetCustomError("The cycle should be between 0 and 7");
+
+    if (!argStream.HasErrors())
```

Comment:
These errors should be changed accordingly (https://github.com/multitheftauto/mtasa-blue/issues/821#issuecomment-522187565).